src/Utilities/TypeTraits.hpp

```diff
@@ -345,6 +345,20 @@ constexpr bool is_unsigned_v = std::is_unsigned<T>::value;
 
 }  // namespace cpp17
 
+/// \ingroup TypeTraitsGroup
+/// C++ STL code present in C++20
+namespace cpp20 {
+/// \ingroup TypeTraits
+template <class T>
+struct remove_cvref {
+  typedef std::remove_cv_t<std::remove_reference_t<T>> type;
```

Comment:
why is this not a using?

Reply:
Because this is how it is specified in the standard